utxo/utxo.go

```diff
@@ -32,12 +31,13 @@ import (
 	ledger_pkg "github.com/xuperchain/xuperunion/ledger"
 	"github.com/xuperchain/xuperunion/pb"
 	pm "github.com/xuperchain/xuperunion/permission"
-	acl "github.com/xuperchain/xuperunion/permission/acl"
+	"github.com/xuperchain/xuperunion/permission/acl"
 	acli "github.com/xuperchain/xuperunion/permission/acl/impl"
 	"github.com/xuperchain/xuperunion/pluginmgr"
 	"github.com/xuperchain/xuperunion/utxo/txhash"
 	"github.com/xuperchain/xuperunion/vat"
 	"github.com/xuperchain/xuperunion/xmodel"
+	"os"
```

Comment:
please notice the import organization separating system packages and 3rd party packages into different parts. so do not place "os" package with 3rd party packages.

Reply:
yep

Reply:
Editor intelligent guide cause this, I fix it